cluster/sync-operator.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+set -ex
+
+kubectl=./cluster/kubectl.sh
+
+function eventually {
+    timeout=$(( $KUBEVIRT_NUM_NODES * 10 ))
+    interval=5
+    cmd=$@
+    echo "Checking eventually $cmd"
+    while ! $cmd; do
+        sleep $interval
+        timeout=$(( $timeout - $interval ))
+        if [ $timeout -le 0 ]; then
+            return 1
+        fi
+    done
+}
+
+function consistently {
```

Comment:
Can we move helper functions to to a place to reuse them ? like sync-common.sh and we source them ?
They would be:
- eventually
- consistently
- isExternal
- isOpenshift
- isDeploymentOk
- push

Reply:
Yes. Good idea. I was actually thinking that as I copied them. :)